frontend/src/app/shared/types/posts/comment-get.model.ts

```diff
@@ -0,0 +1,11 @@
+import {ReactionsGet} from "./reactions-get.model";
+import {UserGet} from "../profiles/user-get.model";
+
+export interface CommentGetModel {
+  commentId: string;
+  content: string;
+  createdAt: Date;
+  modifiedAt: Date | null;
+  author: UserGet;
```

Comment:
ta metoda nie zwróci nam całego usera, tylko najprawdopodobniej dostalibyśmy `firstName`, `lastName`, `userId` i jakieś `avatarUrl`.

Reply:
UserGet aktualnie zwraca tylko te informacje
export interface UserGet {
  userId: string;
  fullName: string;
  profilePhotoUrl: string;
}

Reply:
a to ok